staging/src/k8s.io/csi-translation-lib/go.mod

```diff
@@ -5,7 +5,14 @@ module k8s.io/csi-translation-lib
 go 1.16
 
 require (
+	github.com/coreos/go-oidc v2.2.1+incompatible // indirect
```

Comment:
It doesn't look like your change modifie/added any dependencies. Is this change necessary?

Reply:
I don't think so. The `make verify` has failed and I ran `hack/update-vendor.sh` as was suggested in the error message, I think it just pulled some other dependencies. Would love to get an advice how to fix that

Reply:
Can you try just delete these if these are not used?

Reply:
sure, thanks!